src/components/select/SingleSelectConnected.tsx

```diff
@@ -15,6 +15,7 @@ export interface ISingleSelectOwnProps extends ISelectProps {
     toggleClasses?: string;
     onSelectOptionCallback?: (option: string) => void;
     items?: IItemBoxProps[];
+    togglePrepend?: React.ReactNode;

```

Comment:
why "toggle" prepend?

Reply:
The toggle is the button I guess, maybe buttonPrepend would be clearer